### ISSUE TITLE
Music Storage

### ISSUE THREAD

**Issue by dcamp200** [spotify/android-sdk#131]
Hi there.  I'm noticing the application storage is growing in my app.  It would seem to be consistent with each song as I play it.  (I'm destroying the player when the app shuts down as per the tutorial) Is this normal behaviour?  Is there an api to manage this?  The Spotify app has a option to clear the cache.  
Thanks

**Comment by diedthreetimes** [comment]
:+1:

**Comment by diedthreetimes** [comment]
@dcamp200  did you find a fix for this? I'm still seeing this issue.

**Comment by dcamp200** [comment]
As best I can tell, this is just the way it works(not a bug in my app or the SDK).  I noticed Google Music has the same behaviour, small app but huge growing data.
The only solution I have found is to clear the data using the native Android apis which means loosing local databases etc.. and also logs you out of Spotify.  The Spotify app works like this as well.
It sure would be nice if Spotify had a custom api just to clear the data.  I can see users getting a little freaked out at the growing data.
